tests/tools/onert_train/src/args.cc

```diff
@@ -150,6 +150,8 @@ void Args::Initialize(void)
     }
   };
 
+  auto process_export_path = [&](const std::string &path) { _export_model_filename = path; };
```

Comment:
What happens if the file is already created?

Reply:
At this moment, it just overwrites the existing name of file. We may decide the policy.